java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -148,9 +148,7 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnVector_fromScalar(JNIEnv *env,
     if (row_count == 0) {
       col = cudf::make_empty_column(dtype);
```

Comment:
This is slightly out of scope but (above) `dtype` can be `const` and `cudf::maks_state` can be `auto const`

Reply:
Done. Thx!